### PR TITLE
Add initial dhcp server role

### PR DESCRIPTION
**Pull request by karras**
This role has been migrated from the foreman-ansible repo:

https://github.com/adfinis-sygroup/foreman-ansible/tree/master/ansible/roles/isc_dhcp_server

### CODE REVIEW

File: defaults/main.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+---
+# omapi synchronization port
+isc_dhcp_server_omapi_port: 7911                                                
+                                                                                
+# subnet configuration
+isc_dhcp_server_subnet: []
```
Comment (keachi) [inline defaults/main.yml]: Can you please add an example how to add some subnets here?

File: meta/main.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+
+dependencies: []
+
+galaxy_info:
+  role_name: 'isc_dhcp_server'
+  author: 'Adfinis SyGroup AG'
+  description: ''
```
Comment (keachi) [inline meta/main.yml]: No description?

File: tasks/configuration.yml [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+
+- name: template isc-dhcp-server dhcpd.conf
+  template:
+    src: dhcpd.conf.j2
+    dest: /etc/dhcp/dhcpd.conf
```
Comment (keachi) [inline tasks/configuration.yml:6]: Is that always on the same place or is it better to put this path into a variable?

**Review by keachi — CHANGES_REQUESTED** [review]
Mostly great, but some little changes could make it even better :)

**Comment by hairmare** [comment]
Some changes to the default README.md would also be nice. It should explain why this role exists, what it does and, how to use it.

File: tasks/configuration.yml (hunk 3 above)
Comment (karras) [reply tasks/configuration.yml:6]: So far it's been the same on Ubuntu, Debian, CentOS and RHEL.

**Comment by karras** [comment]
I updated the PR, please review it again. Also used my remaining love to make the README somewhat useful and @hairmare happy.

**Review by keachi — APPROVED** [review]
LGTM

**Pull request closed without merging**